queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
- [ ] Kindly update the query for finding all animals whose name ends in "mon". Your query will find all animals whose name contains "mon".
### Hint: 
```
Update '%mon%' to '%mon'
```